tools/addrundata/main.go

```diff
@@ -0,0 +1,91 @@
+// Command addrundata adds or updates rundata reporting to all tests in the source code,
+// based on each of their README.md.
+//
+// Tests are found under feature/${feature}/${subfeature}/${testkind}/${testname} where
+// testkind is one of "ate_tests", "otg_tests", or simply "tests".  If an ATE test is
+// present, it should have the same rundata as the OTG test.
+//
+// If a test is split into multiple source files, only the source that contains TestMain()
```

Comment:
This seems wrong: the init function is in rundata_test.go and the the test itself isn't modified at all.